docs/SEMAPHORE_GUIDE.md

```
@@ -0,0 +1,352 @@
+mkdir -p ~/.config/containers/semaphoreui/certs
+mkdir -p ~/.config/containers/systemd
+
+# Generate self-signed certificate (10-year)
+openssl req -x509 -newkey rsa:4096 -nodes \
+  -keyout ~/.config/containers/semaphoreui/certs/semaphore.key \
+  -out ~/.config/containers/semaphoreui/certs/semaphore.crt \
+  -sha256 -days 3650 \
+  -subj "/C=MY/ST=Kuala Lumpur/L=Kuala Lumpur/O=Sovereign/OU=IT/CN=localhost"
+```
+
+### B. Install Certificate in Host Trust Store
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)